packages/odyssey-react/src/components/Banner/Banner.module.scss

```diff
@@ -19,28 +19,28 @@
   align-items: center;
   justify-content: center;
   width: 100%;
-  padding-block: $spacing-s;
-  padding-inline: $spacing-m;
-  background-color: cv("blue", "300");
+  padding-block: var(--SpaceRemS);
+  padding-inline: var(--SpaceRemM);
+  background-color: var(--ColorPrimaryLight);
 }
 
 .icon {
   grid-area: icon;
-  margin-inline-end: $spacing-s;
-  font-size: $size-title-5;
+  margin-inline-end: var(--SpaceRemS);
+  font-size: var(--FontSizeHeading5);
```

Comment:
this is where functional names for font sizing falls down a bit as we're using it outside its functional purpose 😞  I worry about this because if we're not disciplined about the application of functional tokens then what is the point of using that nomenclature?

Reply:
cc @edburyenegren-okta let me know your thoughts - we **could** merge what you have here but I'm a bit concerned about that approach going forward as it continues to contradict the functional token name

Reply:
I checked in with Visual and Jose to see how they're managing type sizes (I probably should have started there). Both Figma setups are using this model. Unlike Color, they treat the type size naming more as more representative than functional: "I want *x* to be the size of an H1" rather than "Primary color for primary actions."

Coincidentally, that matches the thinking behind our `<Title>` component props as well -- which I'm hoping most folks will be using rather than referencing type variables directly.

Reply:
I understand, although I do still disagree 😂. Let's get this code merged and we can always revisit discussions about names at a future point. I pray we never reach a point where we need to write CSS like this :joy:

```
.h3 {
  font-size: var(--FontSizeHeading4);
}
```